src/magic_folder/test/__init__.py

```diff
@@ -51,6 +52,11 @@ def _configure_hypothesis():
         settings,
     )
 
+    settings.register_profile(
+        "fast",
```

Comment:
It should be `magic-folder-fast` I think, for the same reason we have `magic-folder-ci` below.  Profile names aren't namespaced in any other way and Hypothesis allows profile name collisions to pass silently, then more or less randomly picks one definition to use when you try to activate the name.

I don't know of any other "fast" profile that's defined but considering how confusing and difficult to debug the situation is I think it's worth being defensive about it.

Maybe I should have written that as a comment above the `"magic-folder-ci"` profile below...

Reply:
ahh, okay, thanks. I added (some of) the above as a comment too